### PR TITLE
Cut release notes for 2.4.3

### PR DESCRIPTION
**Pull request by znicholasbrown**
### Enhancements
- Warn if user tries to login with API key from Cloud 1 — https://github.com/PrefectHQ/prefect/pull/6958
- Improve concurrent task runner performance — https://github.com/PrefectHQ/prefect/pull/6948
- Raise a `MissingContextError` when `get_run_logger` is called outside a run context — https://github.com/PrefectHQ/prefect/pull/6980
- Adding caching to API configuration lookups to improve performance — https://github.com/PrefectHQ/prefect/pull/6959
- Move `quote` to `prefect.utilities.annotations` — https://github.com/PrefectHQ/prefect/pull/6993
- Add state filters and sort-by to the work-queue, flow and deployment pages — https://github.com/PrefectHQ/prefect/pull/6985

### Fixes
- Fix login to private Docker registries — https://github.com/PrefectHQ/prefect/pull/6889
- Update `Flow.with_options` to actually pass retry settings to new object — https://github.com/PrefectHQ/prefect/pull/6963
- Fix compatibility for protected blocks when client/server versions are mismatched — https://github.com/PrefectHQ/prefect/pull/6986
- Ensure `python-slugify` is always used even if [unicode-slugify](https://github.com/mozilla/unicode-slugify) is installed — https://github.com/PrefectHQ/prefect/pull/6955

### Documentation
- Update documentation for specifying schedules from the CLI — https://github.com/PrefectHQ/prefect/pull/6968
- Add results concept to documentation — https://github.com/PrefectHQ/prefect/pull/6992

### Collections
- New [`prefect-hex` collection](https://prefecthq.github.io/prefect-hex/) — https://github.com/PrefectHQ/prefect/pull/6974
- New [`CloudRunJob` infrastructure block](https://prefecthq.github.io/prefect-gcp/cloud_run/) in `prefect-gcp` — https://github.com/PrefectHQ/prefect-gcp/pull/48

### Contributors
* @Hongbo-Miao made their first contribution in https://github.com/PrefectHQ/prefect/pull/6956
* @hateyouinfinity made their first contribution in https://github.com/PrefectHQ/prefect/pull/6955

### CODE REVIEW

File: RELEASE-NOTES.md [hunk 1]
```diff
@@ -1,5 +1,36 @@
 # Prefect Release Notes
 
+## Release 2.4.3
+
+### Enhancements
+- Warn if user tries to login with API key from Cloud 1 — https://github.com/PrefectHQ/prefect/pull/6958
+- Remove single subheading example from main index page — https://github.com/PrefectHQ/prefect/pull/6972
+- Improve concurrent task runner performance — https://github.com/PrefectHQ/prefect/pull/6948
+- Raise a `MissingContextError` when `get_run_logger` is called outside a run context — https://github.com/PrefectHQ/prefect/pull/6980
+- Cache configuration lookups — https://github.com/PrefectHQ/prefect/pull/6959
```
Comment (madkinsz) [inline RELEASE-NOTES.md:10]:
```suggestion
- Adding caching to API configuration lookups to improve performance — https://github.com/PrefectHQ/prefect/pull/6959
```

File: RELEASE-NOTES.md [hunk 2]
```diff
@@ -1,5 +1,36 @@
...
+- Move `quote` to `prefect.utilities.annotations` — https://github.com/PrefectHQ/prefect/pull/6993
+- Add state filters and sort-by to the work-queue, flow and deployment … — https://github.com/PrefectHQ/prefect/pull/6985
```
Comment (madkinsz) [inline RELEASE-NOTES.md:12]:
```suggestion
- Add state filters and sort-by to the work-queue, flow and deployment pages — https://github.com/PrefectHQ/prefect/pull/6985
```

File: RELEASE-NOTES.md [hunk 3]
```diff
@@ -1,5 +1,36 @@
...
+
+### Fixes
+- Fix login to private Docker registries — https://github.com/PrefectHQ/prefect/pull/6889
+- Update `Flow.with_options` to actually pass retry settings to new object — https://github.com/PrefectHQ/prefect/pull/6963
+- Protected block compatibility — https://github.com/PrefectHQ/prefect/pull/6986
+- Ensure python-slugify is always used — https://github.com/PrefectHQ/prefect/pull/6955
```
Comment (madkinsz) [inline RELEASE-NOTES.md:17]:
```suggestion
- Ensure `python-slugify` is always used even if [unicode-slugify](https://github.com/mozilla/unicode-slugify) is installed — https://github.com/PrefectHQ/prefect/pull/6955
```

File: RELEASE-NOTES.md [hunk 4]
```diff
@@ -1,5 +1,36 @@
...
+- Fix configuration caching — https://github.com/PrefectHQ/prefect/pull/7003
```
Comment (madkinsz) [inline RELEASE-NOTES.md:18]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 5]
```diff
@@ -1,5 +1,36 @@
...
+- Add block registration to models init — https://github.com/PrefectHQ/prefect/pull/7008
+
+### Documentation
+- Update docs regarding schedules and the CLI — https://github.com/PrefectHQ/prefect/pull/6968
```
Comment (madkinsz) [inline RELEASE-NOTES.md:22]:
```suggestion
- Add documentation for specifying schedules from the CLI — https://github.com/PrefectHQ/prefect/pull/6968
```

File: RELEASE-NOTES.md [hunk 6]
```diff
@@ -1,5 +1,36 @@
...
+- Fix deployment file name in the tutorial — https://github.com/PrefectHQ/prefect/pull/6956
```
Comment (madkinsz) [inline RELEASE-NOTES.md:23]:
```suggestion
```

File: RELEASE-NOTES.md [hunk 7]
```diff
@@ -1,5 +1,36 @@
...
+- Add results concept to documentation — https://github.com/PrefectHQ/prefect/pull/6992
+
+### Collections
+- Adds prefect hex to catalog — https://github.com/PrefectHQ/prefect/pull/6974
```
Comment (madkinsz) [inline RELEASE-NOTES.md:24]:
```suggestion
- Add [`prefect-hex` collection](https://prefecthq.github.io/prefect-hex/) — https://github.com/PrefectHQ/prefect/pull/6974
```

File: RELEASE-NOTES.md [hunk 8]
```diff
@@ -1,5 +1,35 @@
...
```
Comment (tpdorsey) [inline RELEASE-NOTES.md]:
```suggestion
- Add prefect-hex to Prefect Collections catalog — https://github.com/PrefectHQ/prefect/pull/6974
```

**Review by tpdorsey — APPROVED** [review]

File: RELEASE-NOTES.md [hunk 9]
```diff
@@ -1,5 +1,32 @@
...
+- Fix compatibility for protected blocks when client/server versions are mismatched — https://github.com/PrefectHQ/prefect/pull/6986
+- Ensure `python-slugify` is always used even if [unicode-slugify](https://github.com/mozilla/unicode-slugify) is installed — https://github.com/PrefectHQ/prefect/pull/6955
+
+### Documentation
+- Update documentation for specifying schedules from the CLI — https://github.com/PrefectHQ/prefect/pull/6968
+- Add results concept to documentation — https://github.com/PrefectHQ/prefect/pull/6992
+
+### Collections
+- New [`prefect-hex` collection](https://prefecthq.github.io/prefect-hex/) — https://github.com/PrefectHQ/prefect/pull/6974
+
```
Comment (madkinsz) [inline RELEASE-NOTES.md:25]:
```suggestion
### Collections
- New [`prefect-hex` collection](https://prefecthq.github.io/prefect-hex/) — https://github.com/PrefectHQ/prefect/pull/6974
- New [`CloudRunJob` infrastructure block](https://prefecthq.github.io/prefect-gcp/cloud_run/) in `prefect-gcp` — https://github.com/PrefectHQ/prefect-gcp/pull/48

```

File: RELEASE-NOTES.md (hunk 8 above)
Comment (znicholasbrown) [reply RELEASE-NOTES.md]: @tpdorsey can you submit this suggestion off the latest commit?

Comment (madkinsz) [reply RELEASE-NOTES.md]: Ah yeah I changed this a bit already

**Pull request merged**